tests/test_matchers.py

```diff
@@ -22,6 +22,7 @@
 
 import suricata.update.rule
 from suricata.update import main
```

Comment:
`main` looks like an unused import now. You could instead replace it with `matchers` IMO.